### DISCUSSION TITLE
Impossibilité d'interroger les couches de données

### DISCUSSION THREAD

**Discussion by IGNF-Xavier** [IGNF-Ma-carte/.github#3] [Visualiseur] [answered]
La DREAL Normandie ( identifiant github : jpotel) a testé l'intégration des flux WFS et WMS configurés au travers du swagger de l'API Entrepôt.

L'utilisateur n'a pas trouvé la possibilité sur le résultat de macarte comment interroger les couches d'information ou alors ce dernier n'aurait pas trouvé comment faire le paramétrage pour rendre la couche intégrée interrogeable.

**Comment by Viglino** [comment, marked as answer]
Qu'entendez-vous par "interroger" ?
Il est possible de paramétrer la bulle des objets WFS (comme toute couche vecteur).
Par contre, pour la gestion des bulles en WMS, il faudra attendre la prochaine version (en cours de test)...

**Reply by jpotel** [reply]
Bonjour,
Cela concernait les couches WMS. Il s'agissait d'obtenir à minima une information en cliquant sur l'objet  via l'utilisation d'une requête getfeatureinfo du service WMS.

**Reply by Viglino** [reply]
Ce sera possible dans la prochaine version, visible actuellement sur l'instance de test : https://ignf-ma-carte.github.io/mceditor/
